python-lib/backend/fetch_api.py

```diff
@@ -76,6 +77,56 @@ def get_models():
         return jsonify({'error': str(e)}), 500
     return jsonify(models)
 
+@fetch_api.route("/get_latest_mltask_params", methods=["POST"])
+def get_latest_mltask_params():
+    
+    request_json = request.get_json()
+    current_app.logger.info(f"Recieved request with payload in ml task params: {request_json}")
+    full_model_id = request_json["id"]
+    current_app.logger.info(f"Recieved request for latest params for: {full_model_id}")
+    
+    if is_local:
+        
+        current_app.logger.info(f"Returning Params {dummy_setup_params}")
+        return jsonify(dummy_setup_params)
+    if setup_type != "new":
+        client = dataiku.api_client()
+        mltask = global_DkuMLTask.mltask.from_full_model_id(client,fmi=full_model_id)
+        
+        settings = mltask.get_settings()
```

Comment:
We are not getting the parameters from the selected model. We are getting the parameters from the latest trained model. Is this the behaviour that we want? If so, I think we should not give the option to select a model, it creates some confusion.